app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                f'{visitor["name"]} vaccine is expired.')

```

Comment:
The error message for `NotVaccinatedError` is incorrect. It should indicate that the visitor is not vaccinated, not that the vaccine is expired. Consider changing it to something like: `'{visitor["name"]} is not vaccinated.'`.